exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/profile/QueryStateDisplayName.java

```diff
@@ -0,0 +1,35 @@
+/**
```

Comment:
Please use comment, not Java doc for header.

Reply:
will fix it